cfgov/unprocessed/js/modules/ExternalSite.js

```diff
@@ -0,0 +1,82 @@
+/* ==========================================================================
+   External Site Initialization
+   Used on at least `/external-site/`.
+   ========================================================================== */
+
+'use strict';
+
+var atomicCheckers = require( '../modules/util/atomic-checkers' );
+
+/**
+ * ExternalSite
+ * @class
+ *
+ * @classdesc Initializes a new ExternalSite instance.
+ *
+ * @param {HTMLElement} element DOM Element.
+ */
+function ExternalSite( element ) {
+
+  var BASE_CLASS = 'external-site_container';
```

Comment:
The rationale is something that will not, and more importantly _should not_ be updated during runtime execution of the script. Since this value is an expected value hardcoded in the HTML—to the script it's a constant.